sdk/go/common/workspace/plugins.go

```diff
@@ -802,14 +803,17 @@ func (spec PluginSpec) GetSource() (PluginSource, error) {
 				return newGithubSource(url, spec.Name, spec.Kind)
 			case "gitlab":
 				return newGitlabSource(url, spec.Name, spec.Kind)
+			case "http":
+			case "https":
```

Comment:
Go `case` statements do not fall through automatically.
This should be:
```suggestion
			case "http", "https":
```

If you wanted to run some logic for http and fall through, you'd do:

```go
case "http":
  // stuff
  fallthrough
case "https":
  // more stuff
```

But obviously, that's less readable than `case "http", "https"` in this case.

Reply:
Ah switch rules, how I wish you could all be consistent between the many languages.